wallet/src/service/Offers.ts

```diff
@@ -206,45 +207,95 @@ export const getOfferService = (
     return signSpendAction(offer.spendAction);
   };
 
-  const cancelOffer = _id => {
-    console.log('TODO: cancel offer');
+  const cancelOffer = async (id: number) => {
```

Comment:
let's be consistent in terminology:
```suggestion
  const tryExitOffer = async (id: number) => {
```
"cancel" suggests that it'll always work